benchmark/sparse/linear_regression/inference_benchmark.md

```diff
@@ -0,0 +1,113 @@
+# Linear Regression Benchmark Results 
+
+## Summary
+```
+   Results below show the performance comparison of linear regression with MXNet vs Keras-Tensorflow using sparse tensors
+```                                                   
+
+### Results
+### Inference Benchmark
+### Configuration
+| Dataset          | Synthetic(Randomly generated)                                |
+| :--------------- | :----------------------------------------------------------- |
+| Keras            | v2.2.2                                                      |
+| TensorFlow       | v1.11.0                                                     |
+| MXNet-mkl         | v1.3.0   
+
+###### Using 10 epochs
+#### CPU
+##### Speed
+###### Note speed calculated using an average of 5 runs
+| Instance Type | GPUs  | Batch Size  | Keras-MXNet (Time/Epoch) | Keras-TensorFlow (Time/Epoch)  |
```

Comment:
Why is inference benchmarked as time/epoch? Only training (model.fit) ran several epochs. For inference, you did model.predict(data), the time measured should be the time to run inference on total data. So the smaller the number better performance. We can show in images per second using `time/len(data)`